### PR TITLE
[WIP] don't show popup after copying code

### LINKED ISSUE #31258: Remove "Copied to clipboard" popup
**This is a Feature Request**

**What would you like to be added**

Whenever a user copies to the clipboard using the copy button, a pop message as shown is displayed, which stays on the screen and require the user to manually click ok to close it, this is not an intended behaviour as it requires "manual effort" for not so useful task.
It becomes annoying, especially during the certification exam.

**Why is this needed**
To remove user interaction to manually close the clipboard message, saving time
**Comments**

For eg. try clicking on `pods/storage/pv-claim.yaml Copy pods/storage/pv-claim.yaml to clipboard
` on https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage#create-a-persistentvolumeclaim
Screenshot 2022-01-09 at 1 28 03 PM

** Proposed solution**
It is not required to show the pop message blocking the whole screen, either change the colour of the component for a few seconds, or do nothing at all.

### PR DESCRIPTION
**Pull request by championshuttler**
Drraft to Fixes #31258

### CODE REVIEW

**Comment by mehabhalodiya** [comment]
Our team will surely take a look at this.
I would like to suggest you that please do wait until the particular issue isn't accepted (i.e. `needs-triage`), while you're submitting any PR.
Thank you.

**Comment by sftim** [comment]
/retitle [WIP] don't show popup after copying code

**Review by MdSahil-oss — APPROVED** [review]
Good Work @championshuttler, Hope your changes will work fine.

File: layouts/partials/hooks/body-end.html [hunk 1]
```diff
@@ -34,10 +34,7 @@
         swal("Oh, no…","Sorry, your browser doesn't support copying this example to your clipboard.");
         succeed = false;
     }
-    if (succeed) {
-      swal("Copied to clipboard: ",elem);
-      return succeed;
-    } else {
+    if (!succeed) {
```
Comment (shivanshu1333) [inline layouts/partials/hooks/body-end.html:37]: Instead how about flashing a "copied" message.
For e.g. when you click on the "Copy text" button in this example https://www.w3schools.com/howto/howto_js_copy_clipboard.asp, you'll get a nice "copied" message flashing.

Comment (shivanshu1333) [reply layouts/partials/hooks/body-end.html:37]: Having something like what Tim has mentioned in this comment https://github.com/kubernetes/website/issues/31258#issuecomment-1011040900, would be nice!

**Comment by sftim** [comment]
@championshuttler what are your plans for this PR? We're willing to revise the UI; some indication that copy worked OK would be very nice to keep though.

**Pull request closed without merging**